src/crystal/dwarf/abbrev.cr

```diff
@@ -218,7 +218,7 @@ module Crystal
 
       # Extensions for multi-file compression (.dwz)
       # http://www.dwarfstd.org/ShowIssue.php?issue=120604.1
-      GNurefalt  = 0x1f20
+      Gnurefalt  = 0x1f20
       GnustrpAlt = 0x1f21
```

Comment:
I didn't see any particular reason to capitalize the `N`.

Personally, I'd favor:
```suggestion
      GnuRefAlt  = 0x1f20
      GnuStrpAlt = 0x1f21
```

But...

Reply:
Let's just fix the upper case `N` here, it's very certainly just a typo.
Altering the naming scheme should be a separate change.